src/components/RightPane/CoursePane/CourseRenderPane.js

```diff
@@ -89,17 +91,44 @@ const flattenSOCObject = (SOCObject) => {
     }, []);
 };
 
-const RecruitmentBanner = ({ className }) => (
-    <div className={className}>
-        Interested in web development?
-        <br />
-        <a href="https://forms.gle/v32Cx65vwhnmxGPv8" target="__blank" rel="noopener noreferrer">
-            Join ICSSC and work on AntAlmanac and other projects!
-        </a>
-        <br />
-        We have positions for experienced devs and those with zero experience!
-    </div>
-);
+const RecruitmentBanner = ({ className }) => {
+    // Idk how else to force a function component to update
+    const [, updateState] = React.useState();
+    const forceUpdate = React.useCallback(() => updateState({}), []);
+
+    let displayRecruitmentBanner =
+        !Boolean(window.localStorage.getItem('dismissedRecruitmentBanner')) &&
+        ['COMPSCI', 'IN4MATX', 'I&C SCI', 'STATS'].includes(RightPaneStore.getFormData().deptValue);
```

Comment:
I think this is how you would do it without needing a useCallback function
```suggestion
    const [displayRecruitmentBanner, setDisplayRecruitmentBanner] = useState(
        !Boolean(window.localStorage.getItem('dismissedRecruitmentBanner')) &&
        ['COMPSCI', 'IN4MATX', 'I&C SCI', 'STATS'].includes(RightPaneStore.getFormData().deptValue);
    )
```